io/net/infiniband/ib_bw_perf.py

```diff
@@ -18,14 +18,19 @@
 # ib_write_bw test
 # ib_read_bw test
 # ib_atomic_bw test
+#
+#
+# Based on code by Manvanthara Puttashankar <manvanth@linux.vnet.ibm.com>
+# Copyright: 2015 IBM
+# https://github.ibm.com/ltctest/eio_src/blob/master/bin/infiniband/ib_fvt_test.sh
```

Comment:
I wouldn't call this a big change, would you? (the purpose of authorship is to mark significant changes ) example when the module is completely reworked or extended in a such extend that the original work is not that significant any more. 

See eg. the first response to http://programmers.stackexchange.com/questions/203014/author-has-inserted-copyright-into-code-with-gnu-public-license-notice-implica